### ISSUE TITLE
Cant publish with a mobpack

### ISSUE THREAD

**Issue by SOL0ss** [adam9899/BetterCombat#11]
**Minecraft version** 1.16.5
**Mod loader** - Forge
**Mod version** - Lastest

**Describe the bug:**
- You cannot publish a modpack with this mod installed in it, it says: [
The status of the file (the modpack) has been changed to Rejected

Notes:
The following file versions have **been removed from CurseForge**:
overrides/mods/bettercombat-1.4.5.3+1.16.jar

The modpack will not be approved until these files are replaced with versions that are currently available. ]

The project page was deleted on curseforge and so you cant publish it in a modpack anymore, can you create a new curseforge project to fix this issue please?

**Comment by CaiGuyCrafter** [comment]
I'm having the EXACT same problem. My original problem in the first place is that I was trying to create a server for my friends to play on, since this mod is pretty integral to it, but Aternos doesn't have it since it's not on CurseForge. Then, I had a brilliant idea. I'll just make a modpack with this mod in it, then suggest it to Aternos and it will have the mod inside it. Welp, that didn't work. 
You wanna know why you get that rejection though? Yes, it's because it's not a valid project on CurseForge, (which is stupid by the way because they want every mod you add to be from Curse Forge) but it's actually because the project got taken down by the creators of the original Better Combat, who got pissy over the fact that someone made a 1.16.5 port, even though it's literally in their license agreement to allow people to redistribute and make changes to their mod. It sucks. All I want is to play with my friends, so I decided to make a manual server, but can't even get it to run without crashing before it even starts. 

**TL;DR: It's because of the creators, and the creator of this port likely won't change that any time soon unless he goes into the files, changes the references to the original Better Combat, and advertises it under a completely different name or something. Which probably won't happen. It sucks.**
